src/components/Profile/components/Identification/index.jsx

```diff
@@ -49,11 +50,11 @@ const Identification = ({ profile, myProf, isOnline, createSnackbar }) => {
   // Styles used throughout this component
   const style = {
     button: {
-      background: gordonColors.primary.blue,
+      background: 'var(--mui-palette-primary-main)',
       color: 'white',
```

Comment:
For this component to actually be ready for the dark mode switch, all of these hard-coded `white` colors will also need to be replaced. I'm not saying you need to do that now, but it seems pointless to change only part of a component's styles, and then have to come back to it again later.

Reply:
Yep, those changes will be in the next PR.  Sorry this first one is kind of piecemeal, most of the changes here are just from my experimentation to learn how to do this.  Now that all three of us are intentionally working on whole components, future PRs will be much more organized.

Reply:
No worries, that makes sense.